From: Greg Kroah-Hartman <gregkh@suse.de>
Subject: USB: always announce a device has been added to the system

Distros (like SuSE) want to know this information, to make it easier
to handle support issues.

Odds are no one wants this in mainline, as it clutters up the syslog...

Signed-off-by: Greg Kroah-Hartman <gregkh@suse.de>

---
 drivers/usb/core/hub.c |   10 ++++------
 1 file changed, 4 insertions(+), 6 deletions(-)

--- gregkh-2.6.orig/drivers/usb/core/hub.c
+++ gregkh-2.6/drivers/usb/core/hub.c
@@ -1179,7 +1179,6 @@ void usb_disconnect(struct usb_device **
 	put_device(&udev->dev);
 }
 
-#ifdef DEBUG
 static void show_string(struct usb_device *udev, char *id, char *string)
 {
 	if (!string)
@@ -1187,10 +1186,6 @@ static void show_string(struct usb_devic
 	dev_printk(KERN_INFO, &udev->dev, "%s: %s\n", id, string);
 }
 
-#else
-static inline void show_string(struct usb_device *udev, char *id, char *string)
-{}
-#endif
 
 
 #ifdef	CONFIG_USB_OTG
@@ -1222,7 +1217,10 @@ static int __usb_new_device(void *void_d
 	udev->serial = usb_cache_string(udev, udev->descriptor.iSerialNumber);
 
 	/* Tell the world! */
-	dev_dbg(&udev->dev, "new device strings: Mfr=%d, Product=%d, "
+	dev_info(&udev->dev, "new device found, idVendor=%04x, idProduct=%04x\n",
+		le16_to_cpu(udev->descriptor.idVendor),
+		le16_to_cpu(udev->descriptor.idProduct));
+	dev_info(&udev->dev, "new device strings: Mfr=%d, Product=%d, "
 			"SerialNumber=%d\n",
 			udev->descriptor.iManufacturer,
 			udev->descriptor.iProduct,
